engine/ref_api.h

```diff
@@ -57,6 +57,7 @@ GNU General Public License for more details.
 //    CL_RunLightStyles now accepts lightstyles array.
 //    Removed R_DrawTileClear and Mod_LoadMapSprite, as they're implemented on engine side
 //    Removed FillRGBABlend. Now FillRGBA accepts rendermode parameter.
+// 10. Added R_GetWindowHandle.
 #define REF_API_VERSION 9
```

Comment:
You either add to the changelog _and_ bump the version, or don't mention that in the changelog at all, don't bump the version and just add your changes to the file.

Changelog in file is only for migration from older RefAPI versions, everything else could be and is handled by Git.